src/indexers/apy-snapshots-indexer.ts

```diff
@@ -39,10 +41,26 @@ export async function refreshChainApySnapshots(chain: Chain) {
             mapEntry.thirtyDay += source.thirtyDay;
           }
         });
+
       const mapper = getDataMapper();
       const valueSources = Object.values(sourceMap);
+
+      // check for any emission removal
+      const oldSourcesMap: Record<string, CachedValueSource> = {};
```

Comment:
this is just moving old source removal logic into the same place we do the saves, it makes more sense and makes the other function leaner